libs/ui-lib/lib/common/components/storage/DisksTable.tsx

```diff
@@ -47,7 +48,13 @@ const diskColumns = (showFormat: boolean) => [
   { title: 'Size' },
   { title: 'Serial' },
   { title: 'Model' },
-  { title: 'WWN' },
+  {
+    title: (
+      <>
+        WWN <PopoverIcon bodyContent={'World Wide Name (WWN) is a unique disk identifier'} />
```

Comment:
Needs translation, the text will be propagated to CIM

Reply:
Done.